test/Container/ApplicationFactoryTest.php

```diff
@@ -27,6 +27,68 @@ public function setUp()
         $this->factory   = new ApplicationFactory();
     }
 
+    protected function getContainer($config = null, $router = null, $finalHandler = null, $emitter = null)
```

Comment:
@Maks3w What do think about this approach, of having a helper method for configuring container based on the test specificity?